public/templates/engines_tab.html

```diff
@@ -0,0 +1,31 @@
+<div class="engines tab">
+    <h3>select search engines</h3>
+    <p class="description">
+        Select the search engines from the list of engines that you want results
+        from
+    </p>
+    <div class="engine_selection">
+        <div class="toggle_btn">
+            <label class="switch">
+                <input type="checkbox" class="select_all" onchange="toggleAllSelection()" />
+                <span class="slider round"></span>
+            </label>
+            Select All
+        </div>
+        <hr />
+        <div class="toggle_btn">
+            <label class="switch">
+                <input type="checkbox" class="engine" />
+                <span class="slider round"></span>
+            </label>
+            Duckduckgo
```

Comment:
```suggestion
            DuckDuckGo
```

Reply:
Thanks for the suggestion, but I think we can add it as a new issue with a good-first-issue label, this allows new contributors to get involved in the project and enable them to add value to the project. What do you say??